### ISSUE TITLE
[Bug]: Dark Mode Feature Missing or Not Working

### ISSUE THREAD

**Issue by NishuRajput12** [sayeeg-11/OpenSource-Compass#197]
### Bug Description

The website currently does not have a dark mode option. Users expect a toggle to switch between light and dark themes for better accessibility and reduced eye strain during night time usage.

### Steps to Reproduce

1. Open the website on any device.
2. Look for a dark mode toggle or theme switch.
3. Observe that there is no option available

### Expected Behavior

A visible toggle or button should be present on the navbar or footer to switch between light and dark modes. When toggled, the website colors should invert or switch to a dark-themed palette

### Actual Behavior

No dark mode toggle is available, and the website stays in light mode only.

### Browser

Chrome

### Logs / Screenshots

<img width="1475" height="73" alt="Image" src="https://github.com/user-attachments/assets/7be815df-190d-4ef5-a177-a7e29adb0352" />

<img width="409" height="712" alt="Image" src="https://github.com/user-attachments/assets/59bf552b-4a7e-40f0-a498-ad83664daf66" />

<img width="1881" height="174" alt="Image" src="https://github.com/user-attachments/assets/af5f724b-4d3a-49cf-b359-a5c638e4d6f0" />

**Comment by sayeeg-11** [comment]
> Hi @sayeeg-11 
> I see that this issue has been closed as not planned. However, I’d love to work on adding a dark mode feature and have an implementation approach in mind.
> If this is something you’d consider, may I proceed with a pull request?

@NishuRajput12 

📌 **Project Announcement**

New issue assignments are **temporarily on hold** due to multiple ongoing tasks.

All newly raised issues will be **reviewed and assigned  accordingly once current issues are completed**.  
We appreciate your **patience and cooperation**.

Regards,
 **Project Admin**  
**Sayee Gosavi**
